lms/migrations/versions/396f46318023_events_backfill.py

```diff
@@ -54,7 +59,7 @@ def upgrade():
                 AND grouping.application_instance_id = application_instances.id
                 AND grouping.type = 'course'
        -- Only backfill rows created before we also start tracking them directly on this table
-        WHERE lti_launches.created < '{CUT_OFF_DATE}'
+        WHERE lti_launches.created BETWEEN '{start}' and '{end}'
```

Comment:
Between is inclusive of both extents (https://www.postgresql.org/docs/9.1/functions-comparison.html), so I think this is the same as:

```sql
lti_launches.created >= {start} AND lti_launches.created <= {end}
```

So unless we do something funky with the dates we might end up double counting any records on the boundary.

I think what we want to match what we had before is:

```suggestion
lti_launches.created >= {start} AND lti_launches.created < {end}
```

Reply:
Yup, it might have not affected that many rows over the batches but better be on the safe side with migrations :+1: